pandas/util/mixins.py

```diff
@@ -0,0 +1,28 @@
+"""
+Mixins which can't go to core.common or core.base due to circular imports
+"""
```

Comment:
It can't, it gets a circular import when it is imported in strings.py: 

* `core/base.py` imports `StringMethods`, which can't go to a function: `str = ...(StringMethods,...)`
* `core/strings.py` would then need to import from `core/base.py`

Same with `core/common.py`